docs/en/dev/adapterdev.md

```diff
@@ -316,13 +318,37 @@ Short structure of package.json:
 ```
 - All fields are mandatory. devDependencies should be inside too to enable the grunt tasks.
 
+### Versioning
+
+It is required to use [Semantic Versioning](https://semver.org/) (SemVer) for your adapters.
+
+#### Syntax
+
+Version 1.2.3 =>
+
+| **Major** | **Minor** | **Patch** |
+|-----------|-----------|-----------|
+| 1         | 2         | 3         |
+
+#### Explanation
+
+* Major: Changes which lead to an incompatibility with previous versions
+* Minor: Added or extended functionality  (backwards compatible)
+* Patch: Bug fixes (backwards compatible)
+
+
 ### Deploying
-It is suggested to have the code on github. After the code is stable and lets to install adapter you can share you adapter to other user by asking them to install adapter as follow:
 
+It is recommended to have the adapter code on github. Once your code is stable and lets you install the adapter in your environment successfully, you can share your adapter to other users by one of the following ways:
+
+1. Installing within the ioBroker admin interface: Refer users to section *(5.) Install adapter from your own URL* of [ioBroker Admin documentation](https://github.com/ioBroker/ioBroker.docs/blob/master/docs/en/admin/adapter.md). 
+
+2. Alternatively, ask users to install the adapter through the command line: 
 ```
-npm install https://github.com/yourName/iobroker.adapterName/tarball/master/
+npm install https://github.com/yourName/ioBroker.adaptername/tarball/master/
 ```
-If everything is OK and you have got positive feedback from users you can publish adapter on npm. It would be good if before publishing you will create realease on github.
+
+If everything is OK and you have got positive feedback from users, you can publish the adapter on npm. It would be good if before publishing you will create realease on github.
```

Comment:
typo ... release

Reply:
will fix

Reply:
Not quite sure how I can now update the file in the pull request, I will try and hope not to mess it up :)

Reply:
> Not quite sure how I can now update the file in the pull request, I will try and hope not to mess it up :)

Just edit and commit again in your repository and the same branch, the PR will update automatically.